rotkehlchen/inquirer.py

```diff
@@ -500,6 +509,102 @@ def find_uniswap_v2_lp_price(
         share_value = numerator / total_supply
         return Price(share_value)
 
+    def find_curve_lp_price(
+        self,
+        lp_token: EthereumToken,
+    ) -> Optional[Price]:
+        """
+        1. Obtain the pool for this token
+        2. Obtain prices for assets in pool
+        3. Obtain the virtual price for share and the balances of each
+        token in the pool
+        4. Calc the price for a share
```

Comment:
Write what this returns in success and what in failure. Also if it raises any exceptions (I think not)